airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricQueries.java

```diff
@@ -153,43 +153,63 @@ public static List<Pair<JobStatus, Double>> overallJobRuntimeForTerminalJobsInLa
-                      		AND c.status = 'active'
-                      		AND j.config_type = 'sync'
-                      		AND c.updated_at < now() - interval '24 hours 1 minutes'
-                      		AND cast(c.schedule::jsonb->'timeUnit' as text) = '"hours"'
-                      	GROUP BY 1
-                      	HAVING count(*) < 24 / cast(c.schedule::jsonb->'units' as integer))
-                      UNION (
-                      SELECT
-                      	c.id,
-                      	count(*) as cnt
-                      FROM connection c
-                      LEFT JOIN Jobs j ON j.scope::uuid = c.id
-                      WHERE
-                      	c.schedule IS NOT null
-                      	AND c.schedule != 'null'
-                      	AND j.created_at > now() - interval '1 hours 1 minutes'
-                      	AND c.status = 'active'
-                      	AND j.config_type = 'sync'
-                      	AND c.updated_at < now() - interval '1 hours 1 minutes'
-                      	AND cast(c.schedule::jsonb->'timeUnit' as text) = '"minutes"'
-                      GROUP BY 1
-                      HAVING count(*) < 60 / cast(c.schedule::jsonb->'units' as integer))) as abnormal_sync_jobs
-                      	""";
+                      	left join Jobs j on
+                      		j.scope::uuid = c.id
+                      	where
+                      		c.schedule is not null
+                      		and c.schedule != 'null'
+                      		and j.created_at > now() - interval '24 hours 1 minutes'
+                      		and c.status = 'active'
+                      		and j.config_type = 'sync'
+                      		and c.updated_at < now() - interval '24 hours 1 minutes'
+                      		and cast(c.schedule::jsonb->'timeUnit' as text) = '"hours"'
+                      	group by 1
+                      	having count(*) < 24 / cast(c.schedule::jsonb->'units' as integer)
+                      union
+                      	select
+                      		c.id,
+                      		count(*) as cnt
+                      	from
+                      		connection c
+                      	left join Jobs j on
+                      		j.scope::uuid = c.id
+                      	where
+                      		c.schedule is not null
+                      		and c.schedule != 'null'
+                      		and j.created_at > now() - interval '1 hours 1 minutes'
+                      		and c.status = 'active'
+                      		and j.config_type = 'sync'
+                      		and c.updated_at < now() - interval '1 hours 1 minutes'
+                      		and cast(c.schedule::jsonb->'timeUnit' as text) = '"minutes"'
+                      	group by 1
+                      	having count(*) < 60 / cast(c.schedule::jsonb->'units' as integer) ) as abnormals),
+                      total_sync_jobs as (
```

Comment:
nit: instead of having nested queries, is it possible to break these up? it will make it easier to read and follow.

Reply:
doing two queries is fine

Reply:
Sure, since we are planning to report both percentage and count too.